### PR TITLE
Build Typescript Docker Images

### PR DESCRIPTION
**Pull request by Sushisource**
Fix docker image building, and pass TLS settings through to the TS harness

### CODE REVIEW

File: cmd/build_image.go [hunk 1]
```diff
@@ -143,7 +144,8 @@ func (i *ImageBuilder) buildFromRepo(ctx context.Context) error {
 		return err
 	}
 
-	repoDir := filepath.Join(filepath.Base(tempDir), repoBaseDir)
+	// Not using filepath.Join because we need POSIX path on windows
```
Comment (cretz) [inline cmd/build_image.go:147]: This is what `path.Join` is for

Comment (bergundy) [reply cmd/build_image.go:147]: Oh nice, I didn't know that.

File: cmd/build_image.go [hunk 2]
```diff
@@ -159,7 +161,7 @@ func (i *ImageBuilder) buildFromRepo(ctx context.Context) error {
 			"io.temporal.sdk.repo-url": i.config.RepoURL,
 			"io.temporal.sdk.repo-ref": repoRef,
 		},
-		buildArgs: map[string]string{"SDK_VERSION": repoDir, "REPO_DIR_OR_PLACEHOLDER": repoDir},
+		buildArgs: map[string]string{"SDK_VERSION": fmt.Sprintf("/app/%s", repoDir), "REPO_DIR_OR_PLACEHOLDER": repoDir},
```
Comment (bergundy) [inline cmd/build_image.go:164]: Might as well use `path.Join` here too then.

File: cmd/run_typescript.go [hunk 3]
```diff
@@ -127,10 +143,35 @@ func (r *Runner) RunTypeScriptExternal(ctx context.Context, run *cmd.Run) error
...
+	runArgs := []string{
+		"-r",
+		"tsconfig-paths/register",
+		"./tslib/harness/ts/main.js",
+		"--server",
+		r.config.Server,
+		"--namespace",
+		r.config.Namespace,
+		"--client-key-path",
+		r.config.ClientKeyPath,
```
Comment (bergundy) [inline cmd/run_typescript.go:155]: Looks lie we provide this below

**Review by bergundy — APPROVED** [review]

File: cmd/run_typescript.go (hunk 3 above)
Comment (Sushisource) [reply cmd/run_typescript.go:155]: Thanks, forgot to remove that.

**Pull request merged**